study.md

```diff
@@ -23,41 +23,45 @@ becoming a web developer, this might make a good article.
 In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
-```md
-<!-- your answer here -->
+```md A SPA is a single page application, where all of the updating is handled
+by the client-side using Javascript, instead of loading a new page from the
+server.
+
```

Comment:
Well it's not that the server would be sending pages, its more that you would have to fully re-render an html page when you made an AJAX call or hit a different route. With an SPA you can re-render only parts of a page without have to refresh the entire page.